.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="14" />
+    <bytecodeTargetLevel target="1.8" />
```

Comment:
As you said I should remove the .idea folder, I used it mainly to sync between computers. I'm trying to add some features at the moment, after that's done, I'll try to clean up stuff. Both in code and in project.

In the meantime I would revert this, please.